.travis.yml

```diff
@@ -10,8 +10,8 @@ go:
 go_import_path: github.com/spotahome/redis-operator
 
 before_install:
-  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
-  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.28.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

Comment:
Minikube `v0.28.0` has problems in travis, `v0.25.2` with Kubernetes `1.10.0` version works like a charm

Reply:
Thanks! Changed.